src/Administration/Resources/app/administration/src/module/sw-order/view/sw-order-detail-details/sw-order-detail-details.html.twig

```diff
@@ -79,12 +79,14 @@
         />
         {% endblock %}
 
+        {# @deprecated tag:v6.8.0 - Will be removed, use shipping cost in general view instead. #}
+        {# file: src/Administration/Resources/app/administration/src/module/sw-order/view/sw-order-detail-general/sw-order-detail-general.html.twig #}
         {% block sw_order_detail_details_shipping_costs %}
         <mt-number-field
             v-model="shippingCosts"
```

Comment:
you could also do a `v-if="!feature('v6.8.0.0')` here to be able to show the UI change already with the feature flag